docs/JLBP-6.md

```diff
@@ -176,5 +176,5 @@ Examples in open source
 - JDOM (org.jdom -> org.jdom2)
 - jdeferred (org.jdeferred -> org.jdeferred2)
 
-**Case 5 (Bundle old and new in the existing Maven ID)**
+**Case 5 - Bundle old and new in the existing Maven ID**
```

Comment:
Can we add a noun here? e.g Publish old and new packages with the existing Maven ID

Reply:
Good idea, updated.